packages/vm/src/testing/instance.rs

```diff
@@ -92,6 +92,18 @@ pub struct MockInstanceOptions<'a> {
     pub memory_limit: Option<Size>,
 }
 
+impl MockInstanceOptions<'_> {
+    #[cfg(feature = "stargate")]
+    fn default_features() -> HashSet<String> {
+        features_from_csv("staking,stargate")
+    }
+
+    #[cfg(not(feature = "stargate"))]
+    fn default_features() -> HashSet<String> {
+        features_from_csv("staking")
+    }
```

Comment:
```suggestion
    fn default_features() -> HashSet<String> {
        #[cfg(feature = "stargate")]
        {
            features_from_csv("staking,stargate")
        }
        #[cfg(not(feature = "stargate"))]
        {
            features_from_csv("staking")
        }
    }
```

Reply:
Not very pretty but avoids any mismatch in the function signature

Reply:
They will have the same compile-time checks, and same code.
If there is a mismatch it is only detected under one cfg option.

I find the current version more readable